conf/messages.en

```diff
@@ -465,4 +465,25 @@ case.ruling.status.cancelled = Cancelled
 case.ruling.status.active = Active
 case.ruling.status.expired = Expired
 
-case.ruling.downloadPdf = Download pdf
+case.ruling.downloadPdf = View BTI Ruling
+
+rulingInformation.title = Information on Ruling
+rulingInformation.Heading = Important information about BTI ruling
```

Comment:
We appear to have changed the casing style, lets be consistent with what was already there

Reply:
There's a few different styles used throughout the messages file, but you are right - here there's an inconsistency that means that of the latest entry these don't even all match each other. I'll update to be proper camelcase